packages/react-components/react-list/library/src/components/List/List.cy.tsx

```diff
@@ -173,9 +173,17 @@ const validateSetOfListItems = (expectedStates: Array<boolean>) =>
     listItem.should('have.attr', 'aria-selected', checked.toString());
 
     cy.log('Validate checkbox state on item', index + 1);
-    cy.get(`[data-test="list-item-${index + 1}"] .fui-Checkbox__indicator > svg`).should(
-      checked ? 'exist' : 'not.exist',
-    );
+    // Checkbox now keeps the checkmark SVG in the DOM for motion/animation.
+    // Validate visual state by checking computed opacity (0 -> unchecked, 1 -> checked)
+    cy.get(`[data-test="list-item-${index + 1}"] .fui-Checkbox__indicator > svg`).should($svg => {
```

Comment:
can you import and interpolate css checkbox className here instead of hardcoded string?